lua/cfc_prop_restoration/core/sv/sv_init.lua

```diff
@@ -0,0 +1,102 @@
+local restoration_file_name = "props_backup.txt"
+local recent_disconnects = recent_disconnects or {}
+local prop_data = prop_data or {}
+local expire_time = 300     -- Time (in seconds) for the player to reconnect before data is lost
+local autosave_delay = 60   -- How often (in seconds) the server saves prop data
```

Comment:
i think we could increase the autosave_delay
(maybe even the expiration time as well)